<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirportsDialog</class>
 <widget class="QDialog" name="AirportsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>717</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Аэродромные огни</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Взлет</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Аэродром взлета</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Полоса взлета</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Боковые огни полосы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Рулежные огни аэродрома</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Огни подсвета стоянок аэродрома</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Яркость остальных огней</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QComboBox" name="departureCity">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Moscow DME</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Moscow OST</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>London</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>KarlovyVary</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tivat</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bruxelles</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Venice</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dublin</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Innsbruck</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Keflavic</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Komsomolsk</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pulkovo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sochi</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Toluca</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Moscow SVO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Moscow VKO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Moscow ZIA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Helsinki</string>
            </property>
           </item>
          </widget>
          <widget class="QComboBox" name="departureCode">
           <property name="currentText">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="departure_airport_code"/>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="takeoff_runway_code">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="takeoff_runway_border_lights"/>
        </item>
        <item>
         <widget class="QLineEdit" name="departure_airport_lights_taxiing"/>
        </item>
        <item>
         <widget class="QLineEdit" name="departure_airport_lights_illumination"/>
        </item>
        <item>
         <widget class="QLineEdit" name="departure_airport_other_lights"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Посадка</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Аэродром посадки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Полоса посадки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Боковые огни полосы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Рулежные огни аэродрома</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Огни подсвета стоянок аэродрома</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Яркость остальных огней</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QComboBox" name="arrivalCity">
            <item>
             <property name="text">
              <string>Moscow DME</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>London</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KarlovyVary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tivat</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bruxelles</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Venice</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Dublin</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Innsbruck</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Keflavic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Komsomolsk</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pulkovo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sochi</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Toluca</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Moscow SVO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Moscow VKO</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Moscow ZIA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Helsinki</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="arrivalCode">
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="arrival_airport_code"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="landing_runway_code"/>
        </item>
        <item>
         <widget class="QLineEdit" name="landing_runway_border_lights"/>
        </item>
        <item>
         <widget class="QLineEdit" name="arrival_airport_lights_taxiing"/>
        </item>
        <item>
         <widget class="QLineEdit" name="arrival_airport_lights_illumination"/>
        </item>
        <item>
         <widget class="QLineEdit" name="arrival_airport_other_lights"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Время</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Текущее время в модели</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>78</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="model_simulation_time"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>308</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_pb">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pb">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
